README.md

```diff
@@ -123,6 +124,12 @@ separate directories for each minor version. That soon proved to be a mistake.
 We are keeping the top-level folders in the 1.4 and 1.5 branches so that
 existing users won't be broken.
 
+### Where does the code come from
+
+client-go is synced from https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go. Code changes are made in that location, merged into k8s.io/kubernetes and later synced here.
+
+To help users who need to know from what version of kubernetes the client-go code is copied, we mark the commit that matches kubernetes `vX.Y.Z` tag with `kubernetes-vX.Y.Z` tag.
```

Comment:
Good question. Kubernetes makes patch releases for minor branch (e.g., release-1.6) every 2-4 weeks. As a minor branch gets old, the release frequency decreases. I think there will be too many tags if we match every Kubernetes tags. How about adding the `kubernetes-vX.Y.Z` tags on-request? cc @liggitt